src/utils/walletConnect.ts

```diff
@@ -208,6 +210,31 @@ export async function initListeners() {
 
   client.on('session_proposal', onSessionProposal);
   client.on('session_request', onSessionRequest);
+
+  try {
+    const client_id = await client.core.crypto.getClientId();
+    const token = await getFCMToken();
+    const res = await gretch(`https://wcpush.p.rainbow.me/clients`, {
```

Comment:
so this is just registering the client yah?

Reply:
Yep, all we need atm